lib/elixir/lib/kernel.ex

```diff
@@ -2897,6 +2897,9 @@ defmodule Kernel do
               "invalid write attribute syntax, you probably meant to use: @#{name} expression"
 
       # Typespecs attributes are currently special cased by the compiler
+      name == :typedoc and not bootstrapped?(Kernel.Typespec) ->
+        nil
```

Comment:
```suggestion
        :ok
```

Just to mirror the return from below.

Reply:
I was mirroring the returned value when `Macro` is not bootstrapped 🙂

Reply:
Hahaha. Maybe we should make everything `nil` or `:ok` consistently. But I guess that's in another PR.